### PR TITLE
78 adding permission buttons

### PR DESCRIPTION
**Pull request by natoverse**

### CODE REVIEW

File: app/client/src/modules/common/directives/repodetails/repodetails.js [hunk 1]
```diff
@@ -10,8 +13,27 @@ module.exports = /*@ngInject*/
...
+                function (linkService) {
                     console.log('binding repo details controller', this);
+
+                    this.editLink = links.findByRel('edit-user-permission', this.repo.links);
+
+                    let perm = this.repo.permission;
+                    if (perm) {
+                        this.permission = perm.permissive;
+                        this.github = perm.github;
+                    }
```
Comment (bmathews) [inline app/client/src/modules/common/directives/repodetails/repodetails.js]: Does it end up highlighting the 'none' button?

Comment (bmathews) [inline app/client/src/modules/common/directives/repodetails/repodetails.js]: or do you not want it to?

Comment (natoverse) [inline app/client/src/modules/common/directives/repodetails/repodetails.js]: It does highlight the 'none' button

**Comment by bmathews** [comment]
:shipit:

**Pull request merged**